src/frontend/src/pages/CreateWorkPackagePage/CreateWPForm.tsx

```diff
@@ -3,133 +3,39 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Dispatch, SetStateAction, useState } from 'react';
```

Comment:
can we rename both of these files int his folder to use `WorkPackage` instead of `WP`

every other file uses WorkPackage so when i search for files im not expecting to have to search for WP